src/components/Weathercard.js

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 import "./weather-container.css";
 
-export default function Weathercard (props) {
-    return (
-        <div className="weather-container">
-            <h1>Country: {props.sys.country}</h1>
-            <h1>Temperature: {props.main.temp}</h1>
-            <h2>Pressure: {props.main.pressure}</h2>
-            <h2>Humidity: {props.main.humidity}</h2>
+export default function Weathercard(props) {
+  return (
+    <div className="weather-container">
+      {props.main ? (
+        <div>
+          <h1>Country: {props.sys.country}</h1>
+          <h1>Temperature: {props.main.temp}</h1>
+          <h2>Pressure: {props.main.pressure}</h2>
+          <h2>Humidity: {props.main.humidity}</h2>
         </div>
-    );
-}
\ No newline at end of file
+      ) : (
+        <h1>{props.message}</h1>
```

Comment:
Where do we define the message?

Reply:
Mesaage is there in the prop which we sent. When wrong city is entered it will fetch a create error code and a message as "city not found". It was precise and good so i used it as it is.
![api-error](https://user-images.githubusercontent.com/115643915/196644223-969e65b6-11ec-4fb7-bd2f-413ab96250b3.jpg)